thesis/chapters/1_intro_and_back.tex

```diff
@@ -0,0 +1,118 @@
+\section{Introduction}
+
+Engineers solve problems. Regardless of field, discipline, and generation, every engineer
+strives to solve problems. While the need for problem solvers does not change with time,
+the methods for solving problems certainly do. Consider the process of creating detail 
+drawings. Engineers needed a way to communicate the size, shape, and finish of parts to 
+other engineers, so they hand drew and dimensioned parts. But in the modern day, the 
+thought of hand drafting a detail drawing is laughable. And education has adjusted in
+kind. While students are taught to draw straight lines (which all lines are) and visualize
+without the aid of CAD, no class spends time teaching students how to use a drafting table
+or create detail drawings by hand. Rather, they are taught the basics of visualization
+and then taught how to use SOLIDWORKS.
+
+The same could be said for setting the state in a thermodynamics problem, graphing
+deflection in a load-bearing beam, or finding the Q-value of a fission reaction. At one
+point, looking up values in a table and plugging them into a calculator or using a ruler
+and lined paper were state-of-the-art methods for solving these problems. However, as the 
+times change, so do the methods, and with the advent of programming and its ever lowering 
+barrier to entry, these problems can more easily, and more accurately, be solved with a 
+simple script in any number of programming languages.
+
+Despite this, most classes continue to have students spend hours thumbing through tables
+and solve every problem by hand. In contrast to this, other classes have recognized the
+value of modern solution methods and rely heavily on them, only to find that students
+lack sufficient training to effectively utilize them. Because of this dichotomy, a cohesive 
+and consistent programming ecosystem should be implemented throughout the curriculum 
+of the Mechanical and Nuclear Engineering department.
+
+
+\section{Background}
+
+The first introduction to programming in MNE curriculum is DEN 161: Engineering Problem Solving,
+the first-semester introduction-to-engineering class. The class spends a few weeks introducing
+Python and then a week introducing the Arduino Uno. After this, no programming is used in the
+curriculum until first semester Junior year, as seen in figure \ref{fig:mne_flowchart}. 
+
+\begin{figure}[h]
+    \includegraphics[width=\textwidth]{mne_flowchart}
+    \begin{tabular}{l@{ : }l}
+        Red & Teaches Programming \\
+        Purple & Uses Programming \\
+        Green & Does not utilize Programming, but could \\
+    \end{tabular}
+    \centering
+    \caption{MNE Curriculum Flowchart}
+    \centering
+    \label{fig:mne_flowchart}
+\end{figure}
+
+The next class to use programming is also the only class dedicated to teaching programmig.
```

Comment:
I'm not sure what you're using for writing the text, but check into spell checkers for plain text (one of the downsides of LaTex I suppose)

Reply:
I still remember doing the required spelling bee in 4th grade and losing in the first round on the word "ponies," so my inability to spell is no surprise